Exiled.API/Features/Items/Firearm.cs

```diff
@@ -513,6 +522,41 @@ public void ClearPreferences(IEnumerable<Player> players)
                 ClearPreferences(player);
         }
 
+        /// <summary>
+        /// Spawns the item on the map.
+        /// </summary>
+        /// <param name="position">The location to spawn the item.</param>
+        /// <param name="rotation">The rotation of the item.</param>
+        /// <param name="identifiers">The attachments to be added.</param>
+        /// <param name="spawn">Whether the <see cref="Pickup"/> should be initially spawned.</param>
```

Comment:
Why add a Spawn param on Spawn method?

Reply:
I look this later i don't remember doing this probably was in a PR i have merged

Reply:
Yes, these methods have a lil different logic, it is better to name them `CreatePickup`